utils/eth1.go

```diff
@@ -256,9 +256,9 @@ func FormatAddressLong(address string) template.HTML {
 	}
 	address = FixAddressCasing(address)
 	test := `
-	<span class="text-monospace mw-100"><span class="text-primary">%s</span><span class="text-truncate">%s</span><span class="text-primary">%s</span></span>`
+	<span class="text-monospace mw-100"><span class="text-truncate">%s</span><span class="text-primary">%s</span><span class="text-truncate">%s</span><span class="text-primary">%s</span></span>`
```

Comment:
I'm pretty sure that text-truncate doesn't do anything in a span surrounded by spans that do not have it. Would suggest removing the text-truncate spans.

Reply:
You are right. Removed and tested it, works perfectly fine.